alacritty/src/display.rs

```diff
@@ -345,20 +331,16 @@ impl Display {
     }
 
     /// Update font size and cell dimensions.
-    fn update_glyph_cache(&mut self, config: &Config, font: &Font) {
+    fn update_glyph_cache(&mut self, config: &Config, font: &Font) -> (f32, f32) {
```

Comment:
Can we at least document what type of things this function returns? Since unless you look at a function name you tail call it's not really clear.

Reply:
bump.